src/doc/rustdoc/src/unstable-features.md

```diff
@@ -135,8 +135,24 @@ Book][unstable-masked] and [its tracking issue][issue-masked].
 ## Document primitives
 
 Since primitive types are defined in the compiler, there's no place to attach documentation
-attributes. The `#[doc(primitive)]` attribute is used by the standard library to provide a way to generate
-documentation for primitive types, and requires `#![feature(doc_primitive)]` to enable.
+attributes. The `#[doc(primitive)]` attribute is used by the standard library to provide a way
+to generate documentation for primitive types, and requires `#![feature(doc_primitive)]` to enable.
+
+## Document keywords
+
+This is for rust compiler internal use only.
+
+Rust keywords are documented in the standard library (look for `match` for example).
+
+To do so, the `#[doc(keyword = "...")]` attribute is used. Example:
+
+```rust
+#![feature(doc_keyword)]
+
+#[doc(keyword = "keyword")]
+/// Some documentation about the keyword.
+mod empty_mod {}
```

Comment:
nit: Usually the attribute comes immediately before the type's definition:

```suggestion
/// Some documentation about the keyword.
#[doc(keyword = "keyword")]
mod empty_mod {}
```

Reply:
In the std docs they don't, but I guess it doesn't really matter. I'll update. :)